js/screens/Filtering/FilteringScreen.js

```diff
@@ -154,6 +149,10 @@ const mapStateToProps = state => ({
   sortOptions: state.filter.sortOptions
 });
 
+FilteringScreen.defaultProps = {
+  onPress: {}
```

Comment:
if you do not have propType for onPress then you should not have default prop for onPress.